docs/reference/predicates.md

```diff
@@ -223,6 +223,15 @@ route1: Path("/test") -> "http://www.zalando.de";
 route2: Path("/test") && False() -> "http://www.github.com";
 ```
 
+## Shutdown
+
+Evaluates to true if Skipper is shutting down. Can be used to create customized healthcheck.
```

Comment:
Healthcheck is implemented using two routes and relies on route higher weight when `Shutdown()` is true.
Maybe there are other usecases where users would like to have route enabled only when not in shutdown. For that we could add a companion opposite predicate, e.g. `Running()`.

An alternative naming could be a pair of `Up()` and `Down() == Shutdown()` predicates.

Reply:
I like the pair, even if we do not use it in kubernetes and therefore Up() and Down() might be better names, but I think Shutdown() and Running() are maybe more clear to the user that search for this predicate...

Reply:
Ok, lets have it as is and add the opposite when there is a usecase for it.